channel/pallet/funder_test.go

```diff
@@ -65,7 +65,7 @@ func TestFunder_Timeout(t *testing.T) {
 	// Bob did not fund and times out.
 	wantErr := makeTimeoutErr(1)
 	// Only call Alice's funder.
-	ctx, cancel := context.WithTimeout(context.Background(), 10*s.BlockTime)
+	ctx, cancel := context.WithTimeout(context.Background(), 15*s.BlockTime)
```

Comment:
Maybe use a constant? Also makes it easier to adapt it afterward.

Reply:
Reverted the change. It was actually not a problem.